tools/packaging/packaging.psm1

```diff
@@ -1229,7 +1229,620 @@ function New-ZipPackage
     }
 }
 
+<#
+.SYNOPSIS
+Creates a NuGet package containing unix and windows runtime assemblies.
```

Comment:
> Creates a NuGet package

I think it's not a single NuGet package, right? It should be "create NuGet packages for all PowerShell runtime assemblies".

Reply:
Fixed. And added a list of packages in the description.